src/PositionManager.sol

```diff
@@ -77,6 +100,28 @@ contract PositionManager is
         _executeActions(unlockData);
     }
 
+    /// @inheritdoc INotifier
+    function subscribe(uint256 tokenId, PositionConfig calldata config, address subscriber)
+        external
+        payable
+        onlyIfApproved(msg.sender, tokenId)
+        onlyValidConfig(tokenId, config)
+    {
+        positionConfigs.setSubscribe(tokenId);
```

Comment:
i think a comment here and in unsubscribe could be helpful, like
`// call to _subscribe will revert if the user already has a subscriber`
or something like that?

Reply:
I added in INotifier